### PR TITLE
Update for new ANF API 2019-08-01

### PR DESCRIPTION
**Pull request by leonardbf**
Includes the new python NetApp SDK version 0.7.0
Internal ticket ANF-438

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@MyronFanQiu please help with the PR.

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -30,6 +30,13 @@ Release History
 
 * Fix #11121: When using `az keyvault certificate list`, passing `--include-pending` now doesn't require a value of `true` or `false`
 
+**NetAppFiles**
+
+* Upgrade azure-mgmt-netapp to 0.7.0 to use API version 2019-08-01. This new API version includes:
+
+    -  Some volume properties associated with replication
```
Comment (MyronFanQiu) [inline src/azure-cli/HISTORY.rst]: We don't need these detail information here. 😄

Comment (leonardbf) [reply src/azure-cli/HISTORY.rst]: Ok. I've updated to something more suitable I hope.

Comment (MyronFanQiu) [reply src/azure-cli/HISTORY.rst]: @leonardbf Sorry for my unclear explanation. We need the information "Upgrade azure-mgmt-netapp to 0.7.0". Other part can be decided by yourself. 😄

**Pull request merged**